### ISSUE TITLE
Rebase bugfix-2.0.x to marlins current state

### ISSUE THREAD

**Issue by Nebukadneza** [derhopp/Marlin-with-Anycubic-i3-Mega-TFT#32]
Hi,

I have no anycubic i3 mega, but I’m helping a friend getting this thing to work. I noticed you have replayed your 1.1.x work on bugfix-2.0.x, but on a rather old state. Meanwhile some of the command queueing, serial interface and SD interface was reworked on Marlin side — so rebasing your changes was a bit of a hassle.

As this is a rebase, i can’t open a pull-request. While I realize this is a bit of bad style, it was much easier than merging the current upstream branch back into yours, and this leaves a much cleaner history for eventual integration into upstream.

I also cannot test if my changes work at all — they compile at least :joy: . If you fancy testing them, and maybe fixing them and integrating in your repo, that’d be great.

Here’s what i did:
https://github.com/Nebukadneza/Marlin/commits/derhopp-bugfix-2.0.x-rebase-current

Hope this somewhat helps …

**Comment by Mordaur** [comment]
I was just about to do the same thing. currently i'm using davidramiro's repo, but that's also based on an older version. the vanilla 2.0.1 has some mayor fixes to the trigorilla/ramps pin configuration, and I believe this should be a better candidate for a base config. i tried porting the changes, but got stuck on the gcode/temperature files, as those have also changed alot. and since the anycubic changes are a bit of a hack/hardcode in some places, its hard to follow the code, without a true understanding of the changes. (i'm not a true programmer, perse).
